bundles/statistics/statsgrid2016/components/IndicatorSelection.js

```diff
@@ -130,19 +130,37 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.IndicatorSelection', function (
         indicatorSelector.append(indicDropdown);
         indicSelect.adjustChosen();
 
+        //Regionsets
```

Comment:
Rename variables to be more in line with this being a regionset restriction/filter.